scripts/test/Muon/grouping_tab/grouping_tab_presenter_test.py

```diff
@@ -166,6 +166,66 @@ def test_loading_does_not_insert_invalid_groups(self):
             self.assertEqual(self.grouping_table_view.num_rows(), 1)
             self.assertEqual(self.pairing_table_view.num_rows(), 0)
 
+    def test_loading_selects_all_pairs_if_any_pairs_exist_and_no_default_set(self):
+        self.view.show_file_browser_and_return_selection = mock.Mock(return_value="grouping.xml")
```

Comment:
could we have a helper function to create the groups/pairs? We could even make a function that does the block in the patch (give some of the arguments for the return value).

Reply:
The setup of the groups/pairs is unique in each case here as they describe the outcome of the loading the grouping file so I don't think there is much to be gained from a helper function. I can pull the patch statement out though.